packages/ui/src/components/editor/panels/Scenes/container/index.tsx

```diff
@@ -108,28 +109,40 @@ export default function ScenesPanel() {
               {scenes.map((scene) => (
                 <div
                   key={scene.id}
-                  className="my-2 flex h-[240px] w-[250px] flex-col justify-end rounded-lg bg-theme-surface-main"
+                  className="col-span-2 inline-flex h-64 w-64 cursor-pointer flex-col items-start justify-start gap-2 rounded-lg bg-[#191B1F] p-3 lg:col-span-1"
                 >
                   <img
-                    src={scene.thumbnailURL}
+                    src={scene.thumbnailURL ?? 'static/ir.svg'}
                     alt={scene.key}
                     onError={(e) => {
                       e.currentTarget.src = 'static/ir.svg'
                     }}
                     crossOrigin="anonymous"
-                    className="block h-full grow cursor-pointer self-center rounded-t-lg object-cover"
+                    className="object-fit block h-44 w-full grow cursor-pointer self-center rounded-lg"
                     onClick={() => onClickScene(scene)}
                   />
-                  <div className="flex items-center justify-between px-4 py-1">
-                    <Text className="truncate text-sm leading-5 dark:text-[#A3A3A3]">{getSceneName(scene)}</Text>
+                  <div className="inline-flex items-start justify-between self-stretch px-1">
+                    <div className="inline-flex w-full flex-col items-start justify-start">
+                      <Text className="truncate text-sm leading-5 dark:text-[#A3A3A3]">{getSceneName(scene)}</Text>
+                      <Text
+                        component="h3"
+                        fontSize="xs"
+                        fontWeight="light"
+                        className="h-3.5 w-40 font-['Figtree'] leading-5 text-neutral-100"
+                      >
```

Comment:
```suggestion
                      <Text
                        fontSize="xs"
                        fontWeight="light"
                        className="h-3.5 w-40 leading-5 text-neutral-100"
                      >
```

we have figtree as the default font everywhere